easybuild/easyconfigs/t/tensorflow-compression/tensorflow-compression-2.11.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,55 @@
+name = 'tensorflow-compression'
+version = '2.11.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/tensorflow/compression'
+description = "TensorFlow Compression (TFC) contains data compression tools for TensorFlow."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# Make sure bazel doesn't download TensorFlow by itself.
+local_tf_version = version
+local_tf_dir = 'tensorflow-%s' % local_tf_version
+local_tf_builddir = '%(builddir)s/' + local_tf_dir
+
+sources = [
+    {
+        'source_urls': ['https://github.com/tensorflow/compression/archive/'],
+        'download_filename': 'v%(version)s.tar.gz',
+        'filename': SOURCE_TAR_GZ,
+    },
+    {
+        'filename': 'v%s.tar.gz' % local_tf_version,
```

Comment:
Shouldn't that be:
```suggestion
        'filename': v%(version)s.tar.gz',
```

Reply:
I wanted to keep it separate in case TFC will sometime in the future not require the exact same version number for TF.

Reply:
I though this is the python extension of tensorflow-compression downloaded from PyPi, but it seems to be the tensorflow archive. Might be worth adding the source_urls etc from there in case the downloaded file doesn't exist (anymore)?

BTW: The file `compression_local-tensorflow-repo.sed` and all checksums are missing in this PR.

Reply:
The dependency on TensorFlow sort of guarantees that the tensorflow tar file exists where it should. If it has been removed it should be fixed by refetching TensorFlow I think.